src/integrationTest/java/uk/gov/hmcts/darts/task/runner/impl/AssociatedObjectDataExpiryDeletionAutomatedTaskITest.java

```diff
@@ -595,4 +603,17 @@ private void assignExternalObjectDirectory(TranscriptionDocumentEntity transcrip
                 .createExternalObjectDirectory(transcriptionDocument, objectRecordStatusEnum, externalLocationType, UUID.randomUUID()),
             assignArm, storeArm);
     }
+
+    private void assertAuditEntries(AuditActivity auditActivity, HasIntegerId hasIntegerId, boolean isAnonymized) {
```

Comment:
isAnonymized should be isExpired for consistency

Reply:
This was to follow the same naming conventions as the consuming methods as these have a isDataAnonymised method they call when this flag is set

Reply:
Can you at least change to English spelling instead of American?

Reply:
Actioned